lib/cookie-monster.js

```diff
@@ -16,11 +16,14 @@ var monster = {
     } else {
       valueToUse = encodeURIComponent(value);
     }
-    if (secure){
+    if (secure) {
       secureFlag = "; secure";
     }
+    if (domain) {
+      domain = '; domain=' + encodeURIComponent(domain)
+    }
 
-    document.cookie = name + "=" + valueToUse + expires + "; path=" + path + secureFlag;
+    document.cookie = name + "=" + valueToUse + expires + "; path=" + path + secureFlag + domain;
```

Comment:
If domain is not initiated, will concatenate string 'undefined' to the end of cookie?